events/wc_undead_events.txt

```diff
@@ -1616,6 +1918,55 @@ character_event = {
 	}
 }
 
+#########################################
+# The Transformation into a Wicker Golem
+#########################################
```

Comment:
It doesn't really make sense to create/copy a new event chain. Better relocalize the abomination creation using [custom commands](https://ck2.paradoxwikis.com/Localisation#Custom_commands).